release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -42,6 +42,25 @@ The scope of support for layered and dependent components of {product-title} cha
 
 This release adds improvements related to the following components and concepts.
 
+[id="ocp-4-10-documentation"]
+=== Documentation
+
+[id="ocp-4-10-getting-started"]
+==== Getting started with {product-title}
+
+{product-title} 4.10 now includes a getting started guide. Getting Started with {product-title} defines basic terminology and provides role-based next steps for developers and administrators.
+
+The tutorials walk new users through the web console and the OpenShift CLI (`oc`) interfaces. New users accomplish the following tasks through the Getting Started.
+
+* How to create a project
```

Comment:
Should this be  _Create a project_? I reckon you are trying to mirror the headings in the Getting Started, but it seems odd to have a _How to_ in the list.